crates/nu-cli/tests/format_conversions/html.rs

```diff
@@ -20,12 +20,12 @@ fn out_html_table() {
     let actual = nu!(
         cwd: ".", pipeline(
         r#"
-            echo '{"name": "jason"}' | from json | to html
+            echo '{"name": "darren"}' | from json | to html
```

Comment:
lol

Reply:
This was me trying to make a small change in order to get CI to work again. :)